simulator-core/src/test/java/com/evbox/everon/ocpp/it/security/BasicAuthenticationIt.java

```diff
@@ -0,0 +1,112 @@
+package com.evbox.everon.ocpp.it.security;
```

Comment:
Well i would prefer that the class name matches the use case in OCPP 2.0 exactly, like in other test cases (so I can look at the class name and instantly know it's A02), but this is a minor complaint.